### DISCUSSION TITLE
Welcome to SaikouTV Discussions!

### DISCUSSION THREAD

**Discussion by kemuriXD** [siennaXD/SaikouTV#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.

**Comment by Blitz2224** [comment]
bro can you please bring a new update for the SaikouTV , as kaido server dont have subtitles and it produces really great picture quality and 99% of the animes are available on it too .

**Comment by anasxop** [comment]
please add subtitles in Kaido Server.
